### DISCUSSION TITLE
Support request for MSX computers

### DISCUSSION THREAD

**Discussion by windship** [darthcloud/BlueRetro#667] [General]
Hello, thanks for great project. My old retro consoles becoming modern and really comfortable to use! 

By the way, do you have any plan to support old computers like as MSX?
I hope to use Bluetooth gamepad / keyboard / mouse for it!

I think these links can be kinds of hint, please check and consider to support!

https://msx.gouget.com.br/2014/06/24/ps2-keyboard-to-msx-with-arduino/
http://www.caro.su/msx/kbd4msx.htm

**Comment by darthcloud** [comment]
I'm not familiar with MSX but It's probably just a parallel inteface and as such it's already supported.

grechtech sell calble for it
https://www.tindie.com/products/grechtech/gtrr-levelshifter-modular-5v-blueretro/
